### DISCUSSION TITLE
Removing files from a library?

### DISCUSSION THREAD

**Discussion by Bahlinc-Dev** [llmware-ai/llmware#106] [Q&A]
Not sure how to properly manage a library.    Also, what about replacing a file with an updated version? Does calling add file, add the file again?

**Comment by turnham** [comment]
Thanks for the feedback.  Management and updating of files that are already in a library is on the roadmap.

**Reply by wissamharoun** [reply]
can you comment on how far out this is? thanks

**Comment by doberst** [comment]
@wissamharoun - this is a very old discussion thread - would you mind raising the specific issue in the Issues section - and we can pick up from there?
